app-misc/boxes/boxes-2.2.0.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic toolchain-funcs
+
+DESCRIPTION="Draw any kind of boxes around your text"
+HOMEPAGE="https://boxes.thomasjensen.com/ https://github.com/ascii-boxes/boxes"
+SRC_URI="https://github.com/ascii-boxes/boxes/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86"
+
+RDEPEND"dev-libs/libpcre2[pcre32]"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	sys-devel/bison
+	sys-devel/flex
+"
+
+PATCHES=( "${FILESDIR}/${P}-fix-clang16-build.patch" )
+
+src_prepare() {
+	default
+	append-cflags -Iregexp -I. -ansi -std=c99
```

Comment:
Nit: -Iregexp and -I belong in CPPFLAGS, but that depends on if the build system respects it. It's not wrong to keep it in cflags if not.

Reply:
... but do you actually need this if you're doing CFLAGS_ADDTL? Couldn't you then just do CFLAGS?

I would probably keep this bit out and just use CFLAGS_ADDTL still, as it meddles with internals in the Makefile.

Reply:
This line and `append-ldflags -Lregexp` below are probably obsolete.
There is no longer `regexp` directory in version `2.2.0`:
https://github.com/ascii-boxes/boxes/tree/v1.1.2/src
https://github.com/ascii-boxes/boxes/tree/v2.2.0/src

Reply:
tbh I don't even know why it is in there. Just got that from the previous ebuild. 
But same goes for `append_ldflags` doesn't it?